test/commands/plugins/index.test.ts

```diff
@@ -15,6 +15,21 @@ describe('command', () => {
   .do(output => expect(output.stdout).to.equal('No plugins installed.\n'))
   .it('installs and uninstalls @oclif/example-plugin-ts')
 
+  test
+  .only()
+  .command(['plugins:install', '@heroku-cli/plugin-status'], {reset: true})
```

Comment:
This is highly questionable, as we intend on updating @heroku-cli/plugin-status to no longer be a legacy plugin. I'm not sure where else to find one that will be stable.